### ISSUE TITLE
Generating type adapter for a child (inherited) class

### ISSUE THREAD

**Issue by yringler** [hivedb/hive#119]
**Question**
I have a base class for my data, which has some common data points like title and description. My data classes inherit from it, but the generated type adapter doesn't use the inherited properties. Is there a way to do that? Is that a feature that should be added?

**Code sample**
```dart
class InsideDataBase {
  @HiveField(0)
  final String title;

  @HiveField(1)
  final String description;

  @HiveField(2)
  final List<String> pdf;

  InsideDataBase({this.title, this.description, this.pdf});
}

@HiveType()
class Media extends InsideDataBase {
  @HiveField(3)
  final String source;

  Media({this.source, String title, String description, List<String> pdf})
      : super(title: title, description: description, pdf: pdf);
}
```

**Version**
 - Platform: Android
 - Flutter version: 1.9.1
 - Hive version: 1.1.1

**Comment by yringler** [comment]
Closed as duplicate of #70.

**Comment by leisim** [comment]
Thanks for the report.
This is the next thing I'm going to work on...
